pom.xml

```diff
@@ -60,5 +60,15 @@
 		<artifactId>swagger-jaxrs</artifactId>
 		<version>1.5.22</version>
 	</dependency>
+	<dependency>
+	    <groupId>org.glassfish.jersey.core</groupId>
```

Comment:
what's this dependency for?

Reply:
This is dependency was added to be used in PathParamExceptionMapper.
It's to send a formated exception message for org.glassfish.jersey.server.ParamException.PathParamException class.